src/SourceBuild/content/build.proj

```diff
@@ -16,7 +16,8 @@
                        $(RepoProjectsDir)$(RootRepo).proj"
              Targets="Build"
              BuildInParallel="false"
-             StopOnFirstFailure="true" />
+             StopOnFirstFailure="true"
+             Properties="DotNetBuildOrchestrator=true" />
```

Comment:
We usually don't flow properties via MSBuild unless necessary. I see you want to flow this into dotnet.proj and init-build.proj. Why can't we just set this property globally here in the VMR?

Reply:
You mean pass it in via the script?

Reply:
I can put it in Directory.Build.props if you think that's a better place. I wanted to avoid passing it in as a global property via the script because it's an "always on" sort of property.

Reply:
Yes, global properties should be avoided if the intent is always true. Just set it in D.B.props.